application/cms/views.py

```diff
@@ -511,18 +480,14 @@ def _send_to_review(topic_slug, subtopic_slug, measure_slug, version):  # noqa:
             # If the page was saved before sending to review form's db_version_id will be out of sync, so update it
             measure_version_form.db_version_id.raw_data = [str(measure_version.db_version_id)]
 
-            data_source_form, data_source_2_form = get_data_source_forms(request, measure_version=measure_version)
-
             copy_form_errors(from_form=measure_version_form_to_validate, to_form=measure_version_form)
-            copy_form_errors(from_form=data_source_form_to_validate, to_form=data_source_form)
-            copy_form_errors(from_form=data_source_2_form_to_validate, to_form=data_source_2_form)
 
-            non_form_error_messages = []
-            data_not_uploaded_error = dimensions_not_complete_error = False
+            additional_errors = []
+            data_not_uploaded_error = dimensions_not_complete_error = data_sources_not_added = False
```

Comment:
This line is pretty confusing (with three `=` assignments)...

Reply:
Yep, I don't disagree. 😉

Reply:
🤷‍♂